packages/dapi/lib/grpcServer/handlers/core/getTransactionHandlerFactory.js

```diff
@@ -54,8 +54,16 @@ function getTransactionHandlerFactory(coreRPCClient) {
 
     response.setTransaction(transaction.toBuffer());
     response.setBlockHash(blockHash);
-    response.setHeight(rawTransaction.height);
-    response.setConfirmations(rawTransaction.confirmations);
+
+    // Validate for uint32
+    if (rawTransaction.height >= 0) {
```

Comment:
According to the documentation, this field is optional and contains a height if transaction is minded. So it's nothing or >=1. Why do we check for <-1?

Reply:
Documentation is wrong